doc/source/howtos/upgrade_guide.rst

```diff
@@ -25,6 +27,7 @@ mentioned in these documents when upgrading to |PXC| 5.7.
 
 * `Upgrading from MySQL 5.6 to 5.7 <http://dev.mysql.com/doc/refman/5.7/en/upgrading-from-previous-series.html>`_
 
+
 To upgrade the cluster, follow these steps for each node:
 
 1. Make sure that all nodes are synchronized.
```

Comment:
Make sure that all nodes are synchronized.
Select a node to upgrade. Stop the mysql service on the selected node.